docs/guides/06-introduction-to-animation.md

```diff
@@ -0,0 +1,88 @@
+# Introduction to Animation
+
+Using CARTO VL you can make animated maps of points, lines, and polygons using the [`torque`](https://carto.com/developers/carto-vl/reference/#cartoexpressionstorque) expression. 
+
+_Note: While the expression name torque is inspired by another [CARTO technology](https://carto.com/torque/) for temporal mapping, the two are not equal. When creating temporal maps with CARTO VL, always refer to this documentation._
```

Comment:
To create the yellow note style, type it as: 

**Note:**
While the...

Reply:
Ahhh! ok, awesome will modify